src/main/java/core/basesyntax/MainApp.java

```diff
@@ -1,5 +1,15 @@
 package core.basesyntax;
 
 public class MainApp {
```

Comment:
The MainApp class is well-structured, but make sure to remove any redundant empty lines after the class declaration to keep the code clean.